stl/inc/bitset

```diff
@@ -400,7 +401,18 @@ public:
     }
 
     _NODISCARD bool all() const noexcept {
-        return count() == size();
+        if _CONSTEXPR_IF (_Bits == 0) {
+            return true;
+        }
+        for (size_t _Wpos = 0; _Wpos < _Words; ++_Wpos) {
+            if (_Array[_Wpos] != ~static_cast<_Ty>(0)) {
+                return false;
+            }
+        }
+        if _CONSTEXPR_IF ((_Bits % _Bitsperword) == 0) {

```

Comment:
```suggestion
        if _CONSTEXPR_IF (_Bits % _Bitsperword == 0) {
```

We conventionally don't parenthesize modulo when mixed with equality comparison; modulo has the same precedence as multiplication and we consider that to be well-known higher precedence than equality. (See below for other examples which don't parenthesize.)